File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Assorted/QueryingDoesNotProduceClassMetadataTestCase.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4oUnit;
using Db4oUnit.Extensions;
using Db4objects.Db4o.Tests.Common.Assorted;

namespace Db4objects.Db4o.Tests.Common.Assorted
{
	public class QueryingDoesNotProduceClassMetadataTestCase : AbstractDb4oTestCase
	{
		public class Item
		{
		}

		public virtual void Test()
		{
			Db().Query(typeof(QueryingDoesNotProduceClassMetadataTestCase.Item));
			Assert.IsNull(Container().ClassMetadataForName(typeof(QueryingDoesNotProduceClassMetadataTestCase.Item
				).FullName));
		}
	}
}

File: db4oj.tests/src/com/db4o/db4ounit/common/assorted/QueryingDoesNotProduceClassMetadataTestCase.java
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

package com.db4o.db4ounit.common.assorted;

import db4ounit.*;
import db4ounit.extensions.*;

public class QueryingDoesNotProduceClassMetadataTestCase extends AbstractDb4oTestCase {
	
	public static class Item {
		
	}
	
	public void test(){
		db().query(Item.class);
		Assert.isNull(container().classMetadataForName(Item.class.getName()));
	}

}
